documentation/contributing/contributor-guide.adoc

```diff
@@ -10,6 +10,25 @@ Before you begin you need to:
 
 . xref:setting-up-git[Install and set up Git]
 
+[[contribution-accessibility]]
+== Accessibility
+
+Strimzi aims to make its web content accessible.
```

Comment:
Can we remove the _web_ part? I think that in general we aim to make any content accessible, or?

Reply:
Removed